### PR TITLE
Port Aloe Site to New Frontend Components

### PR DESCRIPTION
**Pull request by zapplecat**
This doesn't do all the things, but a lot of the things to get the original manotomo site up using the most current frontend components.

(I might've forgotten to port the birthday stuff, but we can do that later, as long as it's before Aloe's birthday).

### CODE REVIEW

**Comment by DuongTyler** [comment]
Let's keep working on this branch until we get mobile and games page working instead of merging now

File: frontend/src/components/headerSection/header.css [hunk 1]
```diff
@@ -4,15 +4,16 @@
 }
 
 .app-header {
-    height: calc(15vh + 64px);
+    min-height: 70vh;
```
Comment (Lr-Jim) [inline frontend/src/components/headerSection/header.css:7]: Did you test it on mobiles and portrait monitors?

**Review by Lr-Jim — APPROVED** [review]
lgtm.

**Comment by zapplecat** [comment]
> Let's keep working on this branch until we get mobile and games page working instead of merging now

Lets put games in as a separate PR because that needs a backend change as well. The current backend is missing the `thumbnail` column. Here are the thumbnail images from the old repo https://raw.githubusercontent.com/AreYouW/mano-aloe/fe6895af345165cb1dae53a2a8a9599e79f75cfc/frontend/src/tempGameStash/games.json

**Review by DuongTyler — APPROVED** [review]
lgtm

File: frontend/src/components/headerSection/header.css (hunk 1 above)
Comment (zapplecat) [reply frontend/src/components/headerSection/header.css:7]: Good point with the portrait monitor... It's not great. Let me fix that.

Comment (zapplecat) [reply frontend/src/components/headerSection/header.css:7]: Only 4k portrait view has real problems so I'll file an issue for this and fix later
image

**Pull request merged**